README.md

```diff
@@ -1,131 +1,37 @@
+Build **composable**, **upgradeable**, **secure** Smart Contracts. 
 
-Currently, the focus is on the Soroban VM, but the same ideas apply to other VMs.
+Build frontends that interact easily with (your own or anyone's) Smart Contracts.
 
-## Table of Contents
+Focus on the important parts of your [Soroban](https://soroban.stellar.org/) app. Let Loam handle the tedious bits.
 
-- [Loam SDK](#loam-sdk)
-  - [Table of Contents](#table-of-contents)
-  - [Getting Started](#getting-started)
-    - [Installation](#installation)
-    - [Setup](#setup)
-    - [Redeploy](#redeploy)
-  - [Subcontracts](#subcontracts)
-    - [Creating SubContracts](#creating-subcontracts)
-    - [External API](#external-api)
-  - [Core](#core)
-    - [Using the Core](#using-the-core)
+Loam is comprised of three main components:
+  1. Loam SDK - Create Smart Contracts for Soroban using smaller, more flexible building blocks called "Subcontracts".
+  2. Loam Frontend - Modern frontend tooling paired with a declarative environment configurations to help with local, test, and live blockchain networks.
+  3. Loam CLI - Build and deploy Loam Frontend to work with Smart Contracts.
 
-## Getting Started
+This project is a mono repo containing code primarily for the Loam SDK but also for the Loam CLI, both of which share build code.
 
-### Installation
+This shared build code was created because Smart contracts built with Loam often cannot be built correctly with a standard `cargo build`. They may have complex subcontract interdependencies that need to be resolved in the correct order. So, `loam-build` guarantees that subcontracts get compiled in the correct order. 
 
-To install `just`, run the following command:
+This readme primarily serves to direct you to more specific readme's about the SDK and CLI and examples.
 
-```bash
-cargo install just
-```
+# What is Loam SDK and what is Loam CLI?
 
-### Setup
+The Software Development Kit (SDK) and build tool is for writing smart contracts by creating and assembling "subcontracts". Think of subcontracts as lego blocks you can snap together into a more complex whole. Currently, the SDK is focused on Stellar/Soroban smart contracts, but the subcontract pattern can be made to work with other blockchains as well.
 
-To set up the environment, run:
+The Command Line Interface (CLI) is for building smart contracts authored with Loam SDK. It's also for easily working with smart contract dependencies in a Loam frontend.
 
-```bash
-just setup
-```
 
-### Redeploy
+# Loam SDK and Loam CLI in Depth
+- [Loam SDK](crates/loam-sdk/README.md) - Tool for creating subcontracts.
+  - [loam-subcontract-core](./crates/loam-subcontract-core) - The most basic form of a subcontract, creating an admin/ownsership trait.
+  - [loam-sdk-macro](crates/loam-sdk-macro/README.md) - Code for the `#[subcontract]` macro to create your own brand new subcontract, if existing subcontracts do not suffice.
```

Comment:
This is re-exported by the SDK. In rust procedural macros are a special crate type since they are built and run along side compiling your code. So the pattern is that you have our SDK which is the touch point for users and includes it. So perhaps we could include this in the description of the SDK since it's more of a implementation detail.

Reply:
I think this bulleted list is meant to describe the project structure of this repo. Maybe the heading should be adjusted to say so. And this bullet point can mention that it's re-exported by the main loam-sdk, so there's no reason to depend on this directly.

Reply:
Reorganized the main README